src/DataFixtures/ORM/LoadDonationData.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Donation;
+use AppBundle\Entity\PostAddress;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Ramsey\Uuid\Uuid;
+
+class LoadDonationData extends Fixture
+{
+    /**
+     * Load data fixtures with the passed EntityManager
+     *
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager): void
```

Comment:
please do not change the signature provided by the interface.
`public function load(ObjectManager $manager);`

Reply:
Why ?

Reply:
The goal of an interface is to provide a contract.

Reply:
ok